### ISSUE TITLE
Freezing backbone for finetuning

### ISSUE THREAD

**Issue by james-imi** [ultralytics/ultralytics#13770]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

It says `10` in the `--freeze` but I dont understand whether it is `[10]` or `[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]`?

**Comment by glenn-jocher** [comment]
@james-imi hi there! 👋

Great question about freezing the backbone for fine-tuning. When you specify `--freeze 10`, it means that the first 10 layers of the model will be frozen. This is equivalent to `[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]` (note that it stops at 9, as indexing starts from 0).

Here's a quick example to illustrate:

```bash
yolo train data=coco128.yaml model=yolov8n.pt epochs=100 freeze=10
```

This command will freeze the first 10 layers of the model during training, allowing the rest of the layers to be fine-tuned.

If you have any more questions or need further clarification, feel free to ask! 😊

**Comment by glenn-jocher** [comment]
@djdll when fine-tuning, consider adjusting the learning rate to be lower than initial training to prevent large updates, and focus on freezing early layers if your new data is similar to the original dataset. For OBB tasks, ensure your new data is well-annotated to improve model performance.
